test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/ArrayListTest.scala

```diff
@@ -29,6 +32,73 @@ class ArrayListTest extends AbstractListTest {
     al.ensureCapacity(34)
     al.trimToSize()
   }
+
+  @Test def addAllIndexBounds(): Unit = {
```

Comment:
This one should move up to `CollectionTest`.

Reply:
Thanks! moved to `ListTest` since `def addAll(index: Int, c: Collection[_ <: E])` is defined in `AbstractList`.

Reply:
Ah yes, good.

Small wrinkle: the move was performed in the second commit rather than the first.